src/main/java/tconstruct/weaponry/weapons/Crossbow.java

```diff
@@ -146,13 +144,13 @@ public boolean reload(ItemStack weapon, EntityPlayer player, World world, NBTTag
         tags.setBoolean("Loaded", true);
 
         // remove loaded item
-        if (ammo.getItem() instanceof IAmmo) {
-            if (ammo.hasTagCompound()) {
-                int ammoReinforced = ammo.getTagCompound().getCompoundTag("InfiTool").getInteger("Unbreaking");
-                if (random.nextInt(10) < 10 - ammoReinforced) ((IAmmo) ammo.getItem()).consumeAmmo(1, ammo);
-            }
-        } else {
-            player.inventory.consumeInventoryItem(ammo.getItem());
+        if (!world.isRemote) {
+            if (ammo.getItem() instanceof IAmmo) {
+                if (ammo.hasTagCompound()) {
+                    int ammoReinforced = ammo.getTagCompound().getCompoundTag("InfiTool").getInteger("Unbreaking");
+                    if (random.nextInt(10) < 10 - ammoReinforced) ((IAmmo) ammo.getItem()).consumeAmmo(1, ammo);
+                }
```

Comment:
You could use an enhanced `instanceof` to simplify the code a bit.

```suggestion
            if (ammo.getItem() instanceof IAmmo ammoItem) {
                if (ammo.hasTagCompound()) {
                    int ammoReinforced = ammo.getTagCompound().getCompoundTag("InfiTool").getInteger("Unbreaking");
                    if (random.nextInt(10) < 10 - ammoReinforced) ammoItem.consumeAmmo(1, ammo);
                }
```

You could also collapse the two ifs into one another, if that would be more readable.